.gitignore

```diff
@@ -80,3 +80,7 @@ VERSION.txt
 
 # vim
 *.swp
+
+# main-branch artifacts to explicitly include
+!dash/dash-renderer/build
+!dash/deps
```

Comment:
What's this gitignore change about?

Reply:
Good catch. Not sure where that came from. reverted in 77c3c18